File: be/src/main/java/com/project/popupmarket/dto/payment/ReservationInfoResponse.java
package com.project.popupmarket.dto.payment;

import lombok.Builder;
import lombok.Getter;

import java.math.BigDecimal;

@Getter
@Builder
public class ReservationInfoResponse {
    private String customerKey;
    private String customerName;
    private String customerEmail;
    private String customerTel;
    private String landTitle;
    private String zipcode;
    private String address;
    private String addrDetail;
    private BigDecimal price;
}

File: be/src/main/java/com/project/popupmarket/dto/payment/ReservationInfoResponse.kt
package com.project.popupmarket.dto.payment

import java.math.BigDecimal

data class ReservationInfoResponse(
    val customerId: Long,
    val customerKey: String,
    val customerName: String,
    val customerEmail: String,
    val customerTel: String,
    val landTitle: String,
    val zipcode: String,
    val address: String,
    val addrDetail: String,
    val price: BigDecimal
)